hack/make.sh

```diff
@@ -96,6 +96,9 @@ if [ ! "$GOPATH" ]; then
 	exit 1
 fi
 
+# FIXME Maybe move this elsewhere ?
+DOCKER_BUILDTAGS+=" autogen"
```

Comment:
Probably down in `ORIG_BUILDFLAGS` directly where we have several more that are already statically set unconditionally.